src/components/checkmarkCheckbox/CheckmarkCheckbox.stories.mdx

```diff
@@ -8,11 +8,11 @@ import { FlexContainer } from "../container";
 
 ### CheckmarkCheckbox properties
 
-|      properties | required | type                 | description                                          |
-| --------------: | :------: | :------------------- | :--------------------------------------------------- |
-|       labelText |  false   | string               | Checkbox label                                       |
-|    withoutLabel |  false   | boolean              | Checkbox without label with larger clickable surface |
-|    tooltipProps |  false   | ExtendedTooltipProps | Props for customising the tooltip                    |
+|      properties | required | type                 | description                                                 |
+| --------------: | :------: | :------------------- | :---------------------------------------------------------- |
+|       labelText |  false   | string               | Checkbox label                                              |
+|    withoutLabel |  false   | boolean              | Checkbox without label with larger clickable surface        |
+|          double |  false   | boolean              | If set to true will show double checkmark instead of single |
```

Comment:
You've renamed a prop, right (`tooltipProps` -> `double`)? That's a breaking change then.